packages/runtime/plugin-garfish/src/runtime/utils.ts

```diff
@@ -0,0 +1,13 @@
+declare const __GARFISH_EXPORTS__: string;
+
+export function isRenderGarfish(params?: { appName?: string }) {
+  const renderByGarfish =
+    typeof __GARFISH_EXPORTS__ !== 'undefined' ||
+    (typeof window !== 'undefined' &&
+      window.Garfish &&
+      window.Garfish.activeApps &&
+      window.Garfish.activeApps.some(
+        app => app.appInfo?.name === params?.appName,
```

Comment:
i checked the build products, the 'params' has not been pass to 'isRenderGarfish', do this mean the  ' app.appInfo?.name === params?.appName,' will always be false?
<img width="710" alt="image" src="https://github.com/web-infra-dev/modern.js/assets/65393845/748cda73-ad32-46e8-a7ea-1bce9cb74248">

Reply:
Currently, yes